toc.md

```diff
@@ -42,12 +42,12 @@ This is the table of contents from which the Tanzu Application Platform Document
                 - [Cluster supply chain list](cli-plugins/apps/command-reference/tanzu-apps-cluster-supply-chain-list.md)
         - [Usage and examples](cli-plugins/apps/usage.md)
     - [Authentication and Authorization](authn-authz/overview.md)
-      - [Setting up authentication for Tanzu Application Platform](authn-authz/integrating-identity.md)
-        - [Installing Pinniped on a single cluster](authn-authz/pinniped-install-guide.md)
-      - [Role descriptions](authn-authz/role-descriptions.md)
-        - [Detailed role permissions breakdown](authn-authz/permissions-breakdown.md)
-      - [Bind a user or group to a default role](authn-authz/binding.md)
-      - [Login using Pinniped](authn-authz/pinniped-login.md)
+        - [Setting up authentication for Tanzu Application Platform](authn-authz/integrating-identity.md)
+            - [Installing Pinniped on a single cluster](authn-authz/pinniped-install-guide.md)
+        - [Role descriptions](authn-authz/role-descriptions.md)
+            - [Detailed role permissions breakdown](authn-authz/permissions-breakdown.md)
+        - [Bind a user or group to a default role](authn-authz/binding.md)
+        - [Login using Pinniped](authn-authz/pinniped-login.md)
```

Comment:
I don't think this indent change is necessary because the TOC already looks okay today. Are you seeing issues in how the TOC is rendered?
<img width="375" alt="Screen Shot 2022-04-05 at 10 01 53 AM" src="https://user-images.githubusercontent.com/175788/161771620-7284540c-1827-4ef7-9673-5d99b4b592eb.png">

Reply:
yes there was an issue with indentation and resulting in non grouped topics, which was fixed in the main branch of docs-tap by somebody else in the meantime. https://github.com/pivotal/docs-tap/commit/35f33c450156c61d9c74d426f9a1b937aa6f8343

i think its ok to leave it as is now.